crates/metaslang/bindings/src/builder/functions.rs

```diff
@@ -0,0 +1,49 @@
+use metaslang_cst::KindTypes;
+use metaslang_graph_builder::functions::Functions;
+
+pub fn default_functions<KT: KindTypes + 'static>() -> Functions<KT> {
+    let mut functions = Functions::stdlib();
+    version::add_version_functions(&mut functions);
+    functions
+}
+
+pub mod version {
+    use metaslang_cst::KindTypes;
+    use metaslang_graph_builder::functions::{Function, Functions, Parameters};
+    use metaslang_graph_builder::graph::{Graph, Value};
+    use metaslang_graph_builder::ExecutionError;
+    use semver::{Version, VersionReq};
+
+    pub fn add_version_functions<KT: KindTypes + 'static>(functions: &mut Functions<KT>) {
+        functions.add("version-matches".into(), VersionMatches);
+    }
+
+    struct VersionMatches;
+
+    impl<KT: KindTypes> Function<KT> for VersionMatches {
+        fn call(
+            &self,
+            _graph: &mut Graph<KT>,
+            parameters: &mut dyn Parameters,
+        ) -> Result<Value, ExecutionError> {
+            let version_str = parameters.param()?.into_string()?;
```

Comment:
since all usages are in the form of `version-matches VERSION ....` and using the same value, WDYT of using the `version` supplied here to the binding graph as input, instead of embedding `VERSION` parameter, and round-tripping/re-parsing it every call site? therefore, we don't need the extra parameter.

Reply:
My only hesitation with this is that it may make the rules a tiny bit more opaque. Having the `VERSION` global makes it explicit that we require that. Making this change would allow us to remove the global variable. But the dependency would still be there, only hidden. So, I'm not sure about this. From an efficiency POV makes perfect sense though.

Reply:
I ended up changing this. After some thought, we will have other implicit dependencies when we add the import resolution code. So, this change will make things more consistent.